corehq/apps/reports/standard/monitoring.py

```diff
@@ -1062,8 +1062,8 @@ class WorkerActivityReport(WorkerMonitoringCaseReportTableBase, DatespanMixin):
     is_cacheable = True
 
     fields = [
-        'corehq.apps.reports.dont_use.fields.MultiSelectGroupField',
-        'corehq.apps.reports.dont_use.fields.UserOrGroupField',
+        'corehq.apps.reports.filters.select.MultiGroupFilter',
+        'corehq.apps.reports.filters.users.UserOrGroupFilter',
```

Comment:
That's correct. That's why I've remove all the references to `_all` which was the default placeholder before.